lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -10,16 +10,31 @@ ShopifyApp.configure do |config|
 
   config.reauth_on_access_scope_changes = true
 
-  config.allow_jwt_authentication = <%= !with_cookie_authentication? %>
-  config.allow_cookie_authentication = <%= with_cookie_authentication? %>
-
   config.api_key = ENV.fetch('SHOPIFY_API_KEY', '').presence
   config.secret = ENV.fetch('SHOPIFY_API_SECRET', '').presence
+
   if defined? Rails::Server
     raise('Missing SHOPIFY_API_KEY. See https://github.com/Shopify/shopify_app#requirements') unless config.api_key
     raise('Missing SHOPIFY_API_SECRET. See https://github.com/Shopify/shopify_app#requirements') unless config.secret
   end
 end
 
-# ShopifyApp::Utils.fetch_known_api_versions                        # Uncomment to fetch known api versions from shopify servers on boot
-# ShopifyAPI::ApiVersion.version_lookup_mode = :raise_on_unknown    # Uncomment to raise an error if attempting to use an api version that was not previously known
+Rails.application.config.after_initialize do
```

Comment:
What is the reason that this block needs to run after_initialize?

I understand the instruction in the Upgrading.md document that if this block is run manually, it should do so after initialization because the variables that this context setup uses are from the ShopifyApp configuration which is configured during initialization. But as these lines of code are below the lines that configure ShopifyApp, it would seem that running after initialization is unnecessary.